kpc/management/commands/load_certs.py

```diff
@@ -0,0 +1,337 @@
+
+        self.stdout.write(f'Reading data from {filepath}...')
+        self.rectified_csv = rectify_csv(filepath)
+        self.stdout.write("Newlines converted to '|'")
+
+        self.licensee_id_list = list(
+            Licensee.objects.all().values_list('id', flat=True))
+        self.port_of_export_map = {
+            poe.name: poe.id for poe in PortOfExport.objects.all()}
+
+        self.stdout.write(f"Reading country data from {COUNTRY_CSV}...")
+        self.countries = build_country_map()
+
+        self.load_certs(filepath)
+
+    def get_country(self, value):
+        """Get Country object for incoming country value"""
+        value = ignore_null_str(value)
+        if value:
+            value = self.countries[value]
+            value = Country(code=value)
+        return value
+
+    def load_certs(self, cert_file):
+        self.counter = 0
+        self.test_excluded = 0
+        cert_list = []
+
+        reader = csv.DictReader(self.rectified_csv)
+        for row in reader:
+            self.counter += 1
+            if self.limit and self.counter > self.limit:
+                break
+
+            cert = self.make_certificate(row)
+            if cert:
+                cert_list.append(cert)
+        Certificate.objects.bulk_create(cert_list)
+
+        self.stdout.write(self.style.SUCCESS(
+            f'Processed {self.counter} certificate records.'))
+        self.stdout.write(self.style.SUCCESS(
+            f'Excluded {self.test_excluded} certificates with Licensee: TEST'))
+        self.stdout.write(self.style.SUCCESS(
+            f'Imported {self.counter-self.test_excluded} certificates!'))
+
+    def make_certificate(self, row):
+        """Parse CSV row into Certificate object"""
+        cert = Certificate()
+        cert.number = prepare_number(row['CertNumber'])
+
+        licensee_id = int(row['LicenseeID'])
+        # Ignore TEST(17) licensee records
+        if licensee_id == 17:
+            self.test_excluded += 1
+            return
+        if licensee_id in self.licensee_id_list:
+            cert.licensee_id = licensee_id
+        else:
+            self.stdout.write(self.style.WARNING(
+                f"{cert}: Could not find Licensee ({licensee_id}), setting to None"))
+
+        try:
+            cert.aes = prepare_aes(row['AESNUmber'])
+            if cert.aes:
+                validate_aes(cert.aes)
+        except ValidationError:
+            pass
+            self.stdout.write(self.style.WARNING(
+                f"{cert}: Invalid AES: ({cert.aes}) importing as-is."))
+
+        poe = prepare_poe(row['PortOfExport'])
+        if poe:
+            try:
+                cert.port_of_export_id = self.port_of_export_map[poe]
+            except KeyError:
+                self.stdout.write(self.style.WARNING(
+                    f"{cert}: Unknown Port of Export: ({poe}) setting to None"))
+
+        cert.void = prepare_boolean(row['VoidCert'])
```

Comment:
There are some certs that have a `void` reason, but show up in `prepared` status in the list view.  Examples of this are US17033, US 17126.  I suspect it's because the current site adds a `void reason/date` for certificates that have been `prepared/shipped/delivered` but never moved those statuses to `void`.

Our migration should take care of that for them.  We will keep history of that certificate intact as we are migrating date of issue, ship and delivery.

Reply:
Looks like these get handled by addressing #2 above.

Reply:
Not sure what you mean by addressing #2 above?

Reply:
Whoops, didn't want issue reference there... 

For No. 2 here (https://github.com/USKPA-dev/uskpa/pull/108#pullrequestreview-119973744) 

Setting all certificates with `VoidCert==True` to status Voided, regardless of incoming status from USKPA.